### PR TITLE
IS-65 - Create codeConnects view

### PR DESCRIPTION
**Pull request by zacharysang**
#65: Still needs some work for mobile-friendliness, but wanted to get some feedback on current progress.

Notes:
* Some of the svg files were missing from the bottom 3 icons, so I used what was available and appropriate
* I changed the 'Problems & How We Approach Them' part to make it a lot easier to code (I changed the pointed sides to just make it a rectangle. I can put more time into making this match the mockup, but felt it may require more time than it's worth

Let me know what you think!

### CODE REVIEW

**Comment by bibekg** [comment]
I just changed the base to `programs-page-redesign` since that's what we're gonna put all these Program Views into before merging to master

**Comment by bibekg** [comment]
A few notes:

1. Leave the bar with the text "codeConnects" out of the CodeConnectsView component. Instead we'll put that bar in the OurProgramsPage component and make it dynamic so that it shows the text for whichever view the user is on at the time.
2. I'm seeing a lot of ESLint warnings about variables that are defined but unused. Make sure you clean those errors up.
3. The image behind the text "Through year-long instruction..." is showing up letterboxed with gray bars on the sides. Try using `object-fit: cover` on the image element.

**Comment by zacharysang** [comment]
@bibekg this commit fixes your discussed issues and also improve the mobile support for this view

**Comment by bibekg** [comment]
I think we should also pull out the subtitle bar that says "A diverse approach..." as well since we should include that as part of the OurProgramsPage component as well.

image

Also, this "Problems"/"How We Approach Them" double header is a little bit confusing as a reader. I think it should just be a top header with a larger font size that says "Our Approach" and below it, paragraph-sized text saying something like "The traditional educational model has three key issues that we strive to fix."

image

What do you think?

**Comment by zacharysang** [comment]
Yeah, I think that reads a little more nicely. Doesn't make the reader connect the dots quite as much. Should I go ahead and put it in with my removal of that subheading?

**Comment by bibekg** [comment]
Looks good. Only comment I have now is that the font size hierarchy doesn't make sense. I think "Our Approach" should be larger and the subtitle text should be a little smaller.

**Comment by zacharysang** [comment]
@bibekg I think that the the 'Our Approach' heading should be bigger, but am worried if we make the subheading smaller, it'll look too small.

Here's an image of what is looks like with just the heading upped from $text_md --> $text_lg:

screenshot 2017-12-18 at 10 00 26

**Comment by bibekg** [comment]
I think that's good.

File: src/components/ProgramApproachPiece/ProgramApproachPiece.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/*eslint-disable no-extra-semi */
+import './ProgramApproachPiece.scss';
+import React from 'react';
+
+export default class ProgramApproachPiece extends React.Component {
+    render () {
+        const {
+		        icon,
```
Comment (bibekg) [inline src/components/ProgramApproachPiece/ProgramApproachPiece.js:8]: Spacing

File: src/components/ProgramApproachPiece/ProgramApproachPiece.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+		    </div>
+                    <div className='title'>
+                        {n}) {title}
+                    </div>
+                </div>
+                <div className='section B'>
+                    {desc}
+                </div>
+            </div>
+
```
Comment (bibekg) [inline src/components/ProgramApproachPiece/ProgramApproachPiece.js:29]: Unnecessary empty lines.

**Review by bibekg — CHANGES_REQUESTED** [review]
I'm noticing a lot of odd stylistic issues like odd indentation and empty lines. Do you happen to have ESLint configured for your text editor?

**Comment by zacharysang** [comment]
I don't think I do. I'll see if I can't get that going. What linting style does this project use?

**Review by bibekg — APPROVED** [review]

**Comment by bibekg** [comment]
Gonna approve for now despite the linting issues since we'll get those resolved later in a PR off of `master`.

Just make sure you get the font sizes updated to the most recent screenshot.

**Pull request merged**